io.openems.edge.meter.eastron/src/io/openems/edge/meter/eastron/sdm630/MeterEastronSdm630Impl.java

```diff
@@ -43,14 +43,14 @@
 
 @Designate(ocd = Config.class, factory = true)
 @Component(//
-		name = "Meter.Microcare.SDM630", //
+		name = "Meter.Eastron.SDM630", //
```

Comment:
Please do not change the Factory-ID, as it would lead to a breaking change. See https://community.openems.io/t/neues-meter-sdm-120/2385/18?u=stefan.feilmeier